### DISCUSSION TITLE
Request: Middle-click to close tab

### DISCUSSION THREAD

**Discussion by SteveOnorato** [Able-Bear-Software/tab-nodes-tree#9] [Ideas / Feature Request]
When you middle-click on a tab in the normal Chrome tab bar, it closes the tab.  Can your extension allow the same behavior please?

**Comment by zeroarst** [comment]
Hey @SteveOnorato, that's a great suggestion! I'll definitely look into it. Shouldn't be difficult to implement. I might add an option to enable or disable this feature.

**Comment by zeroarst** [comment]
@SteveOnorato 
Good news! I already implemented it and currently testing on my end. This will be included in 1.2.0.
![image](https://github.com/user-attachments/assets/5096cde1-1f75-4d6d-a6c9-f90d86ca2046)

**Comment by zeroarst** [comment]
Implemented in v1.2.0.
